k8s/chainctl

```diff
@@ -111,7 +112,7 @@ replace-configmaps() {
     kubectl delete configmap $name --ignore-not-found=true >/dev/null
     case "$type" in
       yaml)
-        kubectl create configmap $name --from-file=$file
+        kubectl apply -f $file
```

Comment:
I recall having some problems with this, how are you encoding the name of the configmap in the yaml file?

Reply:
There is the filename that I leave standard (`chainlink-cfg.configmap.yaml`), and then metadata.name.

```
apiVersion: v1
data:
  CHAINLINK_TLS_HOST: acceptance-node.chain.link
  ETH_URL: wss://ropsten.geth.chain.link:8546
kind: ConfigMap
metadata:
  name: chainlink-cfg
```

Reply:
This does leave a warning if the `configMap` was originally made with `create` as opposed to `apply`, but still seems to work. Everything is fine if `apply` is always used.

Reply:
Cool, for some reason it ignored that when I was using it, might have been a bug in k8s I was using or just plain user error.